io.github.igorlogius.scr2ppm.yml

```diff
@@ -0,0 +1,22 @@
+app-id: io.github.igorlogius.scr2ppm
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: scr2ppm
+finish-args:
+  - --socket=x11
+modules:
+  - name: scr2ppm
+    buildsystem: simple
+    build-commands:
+      - ls -l
+      - cc scr2ppm.c -lX11 -lxcb -lXau -lXdmcp -o scr2ppm
+      - install -D scr2ppm /app/bin/scr2ppm
+      - install -D scr2ppm.desktop /app/share/applications/scr2ppm.desktop
+      - install -D scr2ppm-64.png /app/share/icons/hicolor/64x64/apps/scr2ppm-64.png
+      - install -D scr2ppm-128.png /app/share/icons/hicolor/128x128/apps/scr2ppm-128.png
+      - install -D scr2ppm.metainfo.xml /app/share/metainfo/scr2ppm.metainfo.xml
```

Comment:
You need to rename these files. There is `rename-appdata-file`, `rename-desktop-file` and `rename-icon` for that.

What's wrong with using `cmake-ninja` as the buildsystem since there is a `CMakeList.txt` in the upstream repository?

Reply:
This is my first flatpak package,  so sorry, i will likely make a few more mistakes.  i'll take a look at the rename-* directives. 

I did not use cmake, since i did not know if it involed more addional configuration ... and i wanted to keep it as simple as possilble for my first try.  Are there any addional benefits to using it?